app/models/concerns/pxe_loader_suggestion.rb

```diff
@@ -1,7 +1,11 @@
 module PxeLoaderSuggestion
   extend ActiveSupport::Concern
   def suggest_default_pxe_loader
```

Comment:
This method doesn't guarantee that `self.pxe_loader` is set to a string. Was that the intention?

I think this now also no longer returns `self.pxe_loader` at all, unless it is set via the preferred_loader. It doesn't look like that's used so perhaps the API should be formalized to always return nil?

Another difference is that if the user set the pxe-loader parameter to the empty string it will now ignore it and use the preferred loader. I'm not sure if that's common or even valid, but certainly a difference.

Reply:
> This method doesn't guarantee that self.pxe_loader is set to a string. Was that the intention?
Nope, fixed and added test.

> Another difference is that if the user set the pxe-loader parameter to the empty string it will now ignore it and use the preferred loader.

Good point, but how many CUs (if any) did it, hard to say ...